src/components/CertificateTemplates/tlds/sg/edu/smu/degree-cert-template/certificate.js

```diff
@@ -0,0 +1,181 @@
+import PropTypes from "prop-types";
+import { get } from "lodash";
+
+const Template = ({ certificate }) => {
+  // Declaring what variables will be available to the template from the certificate
+  const certificateName = get(certificate, "name");
+  const honors = get(certificate, "additionalData.honors", "\u00a0");
+  const recipientName = get(certificate, "recipient.name");
+  const awardDate = get(certificate, "additionalData.awardDate");
+  const bkgrdImg = get(certificate, "additionalData.bkGrd");
+  const layout = get(certificate, "additionalData.layout");
```

Comment:
We have variant layouts that are dependent on the degree issued, and the layout was designed to be configurable from our data source system. This was to avoid code changes in the template, whenever a new degree is offered. Can we keep this element?

Reply:
different layouts should be reflected as separate templates so this is not possible